### ISSUE TITLE
Core/Command: .character changerace

### ISSUE THREAD

**Issue by CraftedRO** [TrinityCore/TrinityCore#29674]
### Description

.character changerace command does generate some errors first time when you login (on changed character race).

> Player::_LoadSkills: Player 'Test' (GUID Full: 0x000000000000008e Type: Player Low: 142, Race: 1, Class: 6) has forbidden skill 760 for his race/class combination
> Player::_LoadSkills: Player 'Test1' (GUID Full: 0x000000000000008b Type: Player Low: 139, Race: 1, Class: 9) has forbidden skill 753 for his race/class combination

### Expected behaviour

I guess the warnings / errors shoud not appear at all.

### Steps to reproduce the problem

.character changerace Charactername
change his race/name
login on Charactername
check worldserver

### Branch

3.3.5

### TC rev. hash/commit

b4783d6cace49ae0cfe2b0d5059ba776840b307b

### Operating system

Windows 11 x64

### Custom changes

[Passive anticheat](https://github.com/TrinityCore/TrinityCoreCustomChanges/commits/3.3.5-passive_anticheat)

**Comment by Legoso** [comment]
According to https://old.wow.tools/dbc/?dbc=skillline&build=3.3.5.12340#page=5 you can see that skill 760 is Draenei Racial and 753 is Racial - Gnome.
Since both races have got 4 different racial spells, it is difficult to single out which ones have been triggered in your case.

The only conclusion I can draw at the moment is that it looks like the command `.character changerace` fails at removing racial skills when changing race.

**Comment by CraftedRO** [comment]
yea, well the two logs above are just for example, I can reproduce it with every race combination most likely.

**Comment by Legoso** [comment]
Fair enough. Upon further thinking about this issue, is it something that continues to appear in the logs after having changed race, or is it only connected to the initial change before the new selection is saved in the database? I can imagine that maybe Blizzard might have had such errors only when changing, choosing to ignore it, but no more errors after the change is complete. (complete guesswork on my part, it just feels very likely)

**Comment by CraftedRO** [comment]
Probably there's need a switch thing here:
https://github.com/TrinityCore/TrinityCore/blob/8824889f53a9d11d5b943f844900e41456edddec/src/server/game/Handlers/CharacterHandler.cpp#L1785
